src/js/components/reviewData/CertificationModal/CertifyDisclaimer.jsx

```diff
@@ -10,35 +10,32 @@ export default class CertifyDisclaimer extends React.Component {
         return (
             <div className="disclaimer-text" tabIndex={0}>
                 <p>
-                    MPM 2016-03 requires the DATA Act SAO to provide, on a quarterly basis,
-                    reasonable assurance that their agency&apos;s internal controls support the
-                    reliability and validity of the agency account-level and award-level data
-                    reported to USAspending.gov. MPM 2016-03 specifies that this assurance should
-                    leverage data quality and management controls established in statute, regulation,
-                    and Federal-widepolicy and be aligned with the internal control and
-                    risk management strategies in Circular A-123.
+                    MPM 2016-03 requires the DATA Act SAO to provide, on a quarterly basis, reasonable assurance that
+                    their agency's internal controls support the reliability and validity of the agency account-level
```

Comment:
There's an inconsistency between escaping the text or not.

Here it's changed from 
`their agency&apos;s internal` -> `their agency's internal`.

But in `src/js/components/reviewData/CertificationModal/PublishDisclaimer.jsx` Lines 15-16, it's changed from

`my agency’s normal`/`the agency’s last` -> `agency&apos;s normal`/`the agency&apos;s submissions`